doc/asciidoc/algorithms/influence-maximization/celf.adoc

```diff
@@ -0,0 +1,178 @@
+[[algorithms-celf]]
+[.alpha]
+= CELF
+:entity: influenceMaximazation
+:result: spreadGain
+:algorithm: CELF
+
+[abstract]
+--
+This section describes the CELF Influence Maximization algorithm in the Neo4j Graph Data Science library.
```

Comment:
Instead of using CELF in the docs we should use the full name `Cost Effective Lazy Forward Algorithm`
In the first paragraph on this page the abbreviation can be introduced and used for the rest of the page, but the page title etc should be the full name.